src/PrestaShopBundle/Resources/views/Admin/Improve/International/Locations/Country/Blocks/form.html.twig

```diff
@@ -0,0 +1,182 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+{% import '@PrestaShop/Admin/Improve/International/Locations/Country/Macros/address_format_macros.html.twig' as adr %}
+
+{{ form_start(countryForm) }}
+<div class="card">
+  <div class="card-header">
+    {{ 'Countries'|trans({}, 'Admin.Navigation.Menu') }}
+  </div>
+
+  <div class="card-block row">
+    <div class="card-text">
+
+      {{ form_errors(countryForm) }}
+
+      <div class="form-group row">
+        {{ ps.label_with_help(
+          'Country'|trans({}, 'Admin.Global'),
+          'Country name'|trans({}, 'Admin.International.Feature') ~ ' - ' ~ 'Invalid characters:'|trans({}, 'Admin.Global') ~ ' <>;=#{} ',
+          '',
+          '',
+          true
+        ) }}
+        <div class="col-sm">
+          {{ form_widget(countryForm.country) }}
+          {{ form_errors(countryForm.country) }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(
+          'ISO code'|trans({}, 'Admin.International.Feature'),
+          'Two -- or three -- letter ISO code (e.g. "us" for United States).'|trans({}, 'Admin.International.Help'),
```

Comment:
Wouldn't that be more helpful to indicate which ISO code to write?

Reply:
Well doesn't 'e.g. "us"' fulfill this purpose?

Reply:
Oops, sorry Raimondas, I forgot about this comment... Handling ISO codes can be pretty confusing for users, there are many standards depending on the nature of the object designated by the ISO code. So my opinion is that being precise is essential in order to avoid mistakes at this point. Why not write directly _3166-1 ISO code_ here?

Also, I remember I have been told that PrestaShop cannot handle three-letter ISO codes. Can you confirm, @eternoendless? If yes, I guess we should not mention it in the tooltip.

Reply:
Well i have seen multiple instances of three letter ISO codes in BO. Should '3166-1 ISO code' be in 'Admin.International.Help' domain?

Reply:
It depends on whether we decide to include it in the field name (which is _Feature_) or in the tooltip (which is _Help_) that comes with it. About the three-letter ISO codes support or not, let's wait for @eternoendless's feedback before, I would like to be sure about it.
🙂

Reply:
@PrestaShop/prestashop-core-developers, any recommendation?

Reply:
🆙

Reply:
@eternoendless 😛

Reply:
PrestaShop only supports two-letter country iso codes AFAIK, and it also mixes up language codes and country codes (yay!).

Reply:
I think we're talking about "ISO 3166-1 alpha-2" codes here.

Reply:
@matks, @eternoendless, thanks a lot for your feedback! In order to avoid confusion here, I recommend writing: 

_ISO 3166-2 code_ in _Admin.International.Feature_
_Two-letter country ISO code (e.g. "US" for the United States)_ in _Admin.International.Help_